sagemaker-core/spark-processing-sm-core.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<p>nit: clear the cell outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Done